### ISSUE TITLE
Did you include adv_loss in the overall loss function?

### ISSUE THREAD

**Issue by ZoeyZheng0** [BoomShakaY/AIM-CCReID#12]
Hi, thank you for submitting the training code!

I am trying to employ your code in my pipeline, but I have a small doubt. Did you include the adv_loss in the overall loss function?

You only mentioned Cloth Loss, ID Loss, CAL Loss, and KL Loss in the paper. While in the train.py file, 
`loss = cla_loss + adv_loss + config.LOSS.PAIR_LOSS_WEIGHT * pair_loss`
adv_loss is also included. From other issues, I found that pair_loss_weight should be settled to zero, but I did not find any discussion about the weight of adv_loss.

Shall I delete adv_loss while computing, or do I misunderstand the loss function?

**Comment by BoomShakaY** [comment]
I assume your question is about whether to use pair_loss and adv_loss in the training.

According to the paper, there are no pair_loss, thus I set the weight of pair loss to 0. However, for larger datasets such as LaST and DeepChange, you can consider increasing the pairloss weight, which is optional.
As for the adv_loss, yes, the adv loss is also mentioned in the paper, the default weight is 1. You can adjust the weight of adv_loss according to your dataset. 

The details are aligned with the paper, carefully reading the paper might solve your problem, thanks.
